Makefile

```diff
@@ -25,7 +25,7 @@ xcop:
 	xcop $(find . -name '*.xml')
 
 flake8:
-	python3 -m flake8 aibolit test scripts setup.py
+	python3 -m flake8 aibolit test scripts setup.py --exclude scripts/target/*
```

Comment:
As i understand, folder `scripts/target` stores third party repositories. They might be not be flake8 compatible. At least after running `make samples` for the scripts makefiles and was not able to run `make flake8` on the main makefile because of errors in `scripts/target`  folder files.

Reply:
you do smth wrong, everything works correctly, need to discuss. `Scripts/target` is not third party repositories. `Scripts/target` folder has scripts which gathers dataset and is closely related to main framework

Reply:
Check the error log for flake8
https://pastebin.com/PmSyXe01